db/seeds.rb

```diff
@@ -18,6 +18,13 @@ def format_errors(model = nil)
     ForemanTasks.sync_task(::Actions::Katello::Organization::Create, org) unless org.library
     User.current = nil
   end
+  if Setting['db_pending_seed']
+    admin = User.where(:login => ENV['SEED_ADMIN_USER'].present? ? ENV['SEED_ADMIN_USER'] : 'admin').first
+    if admin && admin.default_organization.nil?
+      admin.default_organization = Organization.find_by_name!(ENV['SEED_ORGANIZATION'])
+      admin.save!
+    end
+  end
```

Comment:
if Setting['db_pending_seed'] was the intended way to prevent this.  I'm not sure how find_or_create_by_login would help here (since we aren't creating the user).